### PR TITLE
test to verify behaviour with EdDSA certificates

### PR DESCRIPTION
**Pull request by tomato42**
### Description

Verify that Ed25519 and Ed448 can be used in server's certificate.

### Motivation and Context

Test coverage for the EdDSA keys

### Checklist

- [x] I have read the [CONTRIBUTING.md](https://github.com/tlsfuzzer/tlsfuzzer/blob/master/CONTRIBUTING.md) document and my PR follows [change requirements](https://github.com/tlsfuzzer/tlsfuzzer/blob/master/CONTRIBUTING.md#change-requirements) therein
- [ ] the changes are also reflected in documentation and code comments
- [x] all new and existing tests pass (see CI results)
- [ ] [test script checklist](https://github.com/tlsfuzzer/tlsfuzzer/wiki/Test-script-checklist) was followed for new scripts
- [ ] new test script added to `tlslite-ng.json` and `tlslite-ng-random-subset.json`
- [x] new and modified scripts were ran against popular TLS implementations:
  - [x] OpenSSL
  - [ ] NSS
  - [ ] GnuTLS
- [ ] required version of tlslite-ng updated in requirements.txt and README.md

- - -
This change is [Reviewable](https://reviewable.io/reviews/tlsfuzzer/tlsfuzzer/977)

### CODE REVIEW

File: scripts/test-tls13-eddsa.py [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+        else:
+            raise ValueError("Unknown option: {0}".format(opt))
+
+    if args:
+        run_only = set(args)
+    else:
+        run_only = None
+
+    if not ciphers:
+        ciphers = [CipherSuite.TLS_AES_128_GCM_SHA256]
```
Comment (The-Mule) [inline scripts/test-tls13-eddsa.py:104]: Is CipherSuite.TLS_EMPTY_RENEGOTIATION_INFO_SCSV no longer needed in ciphers?

File: scripts/test-tls13-eddsa.py [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+                    print("OK-expected failure\n")
+        else:
+            if res:
+                good += 1
+                print("OK\n")
+            else:
+                bad += 1
+                failed.append(c_name)
+
+    print("Basic communication test with TLS 1.3 server")
```
Comment (The-Mule) [inline scripts/test-tls13-eddsa.py:281]: Maybe we can mention here that ths is to test EdDSA actually?

**Review by The-Mule — APPROVED** [review]
LGTM (the two nitpicks mentioned are not really anything blocking I think).

File: scripts/test-tls13-eddsa.py (hunk 2 above)
Comment (tomato42) [reply scripts/test-tls13-eddsa.py]: good point, fixed, please check the new description

**Review by The-Mule — APPROVED** [review]

File: scripts/test-tls13-eddsa.py (hunk 2 above)
Comment (The-Mule) [reply scripts/test-tls13-eddsa.py]: Thanks!

File: scripts/test-tls13-eddsa.py (hunk 1 above)
Comment (The-Mule) [reply scripts/test-tls13-eddsa.py:104]: Ah, good then, thanks for clarification.

**Pull request merged**